### DISCUSSION TITLE
Comparison to other projects

### DISCUSSION THREAD

**Discussion by jcbhmr** [devcontainers-community/.github#7] [General]
**TL;DR:** @devcontainers-community is a **community-driven** project focused on **high-quality** features & templates that are for **popular tools** while @devcontainers-contrib appears to be a **feature-only** project with a single maintainer

<table><td>@devcontainers-community<td>@devcontainers-contrib<tr><td>

- currently 2 maintainers, hopefully more! ❤️ open to anyone 😀
- focused on making **things that enhance the official tools**
- use standard mostly-official workflows and conventions

<td>

- one maintainer
- lots of awesome features
- some cool custom tooling to generate features 😎
- sometimes `$TOOL-sdkman` and `$TOOL-asdf` or other suffixes for multiple ways to install tool `$X` 🎉

</table>

the differences:
- number of people involved
- level of custom workflow -ness
- conventions

as for the relation to the official @devcontainers project, neither of these projects @devcontainers-community nor @devcontainers-contrib are in any way related to the official @devcontainers project, but their participants can and do engage in issues and other ways with the official @devcontainers project.

⚠ AS OF NOW there's only two maintainer members of the top-level @devcontainers-community organization: @jcbhmr and @mering .  this hopefully will be expanded to include more people in the future! want to be one of said people? [join!](https://github.com/orgs/devcontainers-community/discussions/new?category=general)

**Comment by jcbhmr** [comment]
heres another good outline of what _me thinks_ this project is intended to be:
https://github.com/devcontainers-community/.github id love to hear ideas for better mission outlines

**Comment by eitsupi** [comment]
@jcbhmr Shouldn't it be stated that there is currently only one maintainer for this project and that you alone are making all the decisions? (i.e. I assume you immediately discarded some of the things I built and rewrote everything to be your own as soon as I left this organization)

I am frazzled, especially regarding the rewrite of the dart Feature, but I hope someone in the dart community will work on it, since I don't use dart.

**Reply by mering** [reply]
I recently joined with https://github.com/devcontainers-community/features-bazel (only that repo not the org).

Would be really great to have more maintainers of the overall org to not depend on single persons being available. I think it is great that feature maintainers have rights in the corresponding repos to not block them updating the features. For the general org, I think about 5 maintainers would be great. If additional people join the org, @eitsupi would you be open to join back?

UPDATE: I now also joined the org as member.

**Reply by jcbhmr** [reply]
> @jcbhmr  Shouldn't it be stated that there is currently only one maintainer for this project and that you alone are making all the decisions? (i.e. I assume you immediately discarded some of the things I built and rewrote everything to be your own as soon as I left this organization)

Yep! 👍 updated the discussion to warn about that particular issue.

i did discard some things and completely re-architect the way the repos were structured (monorepo => polyrepo). i figured this would be good to get people onboard who would be able to help with a specific feature without being overwhelmed by such a large monorepo.

hopefully another person besides me can help make such big decisions so I'm not running amok making unilateral decisions that could be bad. 😬 i dont want to do that, and i know making the monorepo=>polyrepo move was a sign of single-person-decision happening and i dont think that's good without discussion. hence, all the "join the org" adverts 🤣

> I am frazzled, especially regarding the rewrite of the dart Feature, but I hope someone in the dart community will work on it, since I don't use dart.

i can understand your frazzlement. i did make a ton of changes that had been stewing in my brain. particularly the monorepo=>polyrepo idea.

as for the Dart feature https://github.com/devcontainers-community/features-dart-sdk i deliberately tried to copy one of the features from https://github.com/search?q=dart+feature&type=registrypackages specifically https://github.com/FoodieAppServices/vsc-features and then [discuss with them](https://github.com/devcontainers-community/features-dart-sdk/issues/1) how to make it better. that seems like the best way to get the Dart expertise needed to maintain such a Dart feature when I don't have said Dart expertise. now that it's there and the discussion's open, you're right i should bring you into the conversation with the original https://github.com/devcontainers-community/features/blob/8522fe1bf77b027373dbf10b332010599afc0633/src/dart/install.sh that you authored.

**Reply by mering** [reply]
A small clarification that I am only an org _Member_ and no _Owner_ (I think these are the two default org roles).

**Reply by jcbhmr** [reply]
oh whoops, you've been upgraded 🚀

**Reply by eitsupi** [reply]
Thanks @mering,

> If additional people join the org, @eitsupi would you be open to join back?

I don't know that at this point. I have really lost my energy.
